examples/gno.land/p/demo/paginate/paginate.gno

```diff
@@ -12,22 +19,116 @@ type renderer interface {
 // TODO this function belongs somewhere convenient.
```

Comment:
btw, a proper place would be a new `gno.land/p/demo/md` package with such generic helpers related to markdown.

feel free to move it right now.